src/dialogs/more-info/controls/more-info-alarm_control_panel.html

```diff
@@ -97,25 +93,27 @@
 
   stateObjChanged(newVal, oldVal) {
     if (newVal) {
-      this.codeFormat = newVal.attributes.code_format;
-      this.codeInputVisible = this.codeFormat !== null;
-      this.codeInputEnabled = (
+      const props = {
+        codeFormat: newVal.attributes.code_format,
+        armHomeButtonVisible: newVal.state === 'disarmed',
+        armAwayButtonVisible: newVal.state === 'disarmed',
```

Comment:
we also have armCustomBypass

Reply:
... and night https://github.com/home-assistant/home-assistant-polymer/pull/760

Reply:
Out of scope for this PR. This is just translating it to use `setProperties`. I made no changes to the logic.